nelc/frontend-essentials
# Jlc/coursename feedback card

# Description
Add course name extra field in the feedback card.
Also the feedback carrousel logic is updated with this test.

## How to test
Clone the branch and render the component. Check that now if the API hast the relationship related with `course_id.display_name...`
Check the course name is rendered.

## After
![Peek 2023-07-11 15-53](https://github.com/nelc/frontend-essentials/assets/51926076/bbe4ddc1-1fe8-42f1-b334-4571d6c2c1c2)

## extra information
**related jira story**
https://edunext.atlassian.net/browse/FUTUREX-460?atlOrigin=eyJpIjoiMzkyYjQ0ZDM3MTQzNDQ0NDlkOWI5MDI3ZjBiNjBiMjkiLCJwIjoiaiJ9
